docs/source/api/image.rst

```diff
@@ -71,7 +71,7 @@ _____________________
     ~instance_segmentation.model.InstanceSegmentation
     ~instance_segmentation.data.InstanceSegmentationData
 
-    instance_segmentation.data.InstanceSegmentationInputTransform
+    ~instance_segmentation.data.InstanceSegmentationInputTransform
```

Comment:
Tilde would only be needed if you can do:
```
from flash.image import InstanceSegmentationInputTransform
```
Otherwise it should be:
```suggestion
    instance_segmentation.data.InstanceSegmentationInputTransform
```